pages/linux/screenkey.md

```diff
@@ -0,0 +1,32 @@
+# screenkey
+
+> A screencast tool to display your keys.
+> More information: <https://www.thregr.org/~wavexx/software/screenkey/>.
+
+- Display your keys pressed on the screen:
+
+`screenkey`
+
+- Display your keys and mouse buttons pressed on the screen:
+
+`screenkey --mouse`
+
+- Launch settings menu of screenkey:
+
+`screenkey --show-settings`
+
+- Change the postion(top/center/bottom/fixed) of screenkey:
+
+`screenkey --position top`
+
+- Change the modifiers displayed on the screen like ctrl to ⌘ in mac mode:
+
+`screenkey --mods-mode mac`
```

Comment:
```suggestion
- Change the format of the key modifiers displayed on screen:

`screenkey --mods-mode {{normal|mac|emacs}}`
```

Reply:
I added all the available options below. Should I commit all the suggested  changes as a single commit if everything looks good?
```suggestion
- Change the format of the key modifiers displayed on screen:

`screenkey --mods-mode {{normal|emacs|mac|win|tux}}`
```